libnetwork/drivers/bridge/setup_ip_tables_linux_test.go

```diff
@@ -92,6 +93,11 @@ func createTestBridge(config *networkConfiguration, br *bridgeInterface, t *test
 	if err := setupBridgeIPv4(config, br); err != nil {
 		t.Fatalf("Failed to bring up the testing Bridge: %s", err.Error())
 	}
+	if config.EnableIPv6 {
+		if err := setupBridgeIPv6(config, br); err != nil {
```

Comment:
Given that we're passing `config` to the function , would it make sense to sense to have `setupBridgeIPv6` return early if `EnableIPv6` is not set?


```go
func setupBridgeIPv6(config *networkConfiguration, i *bridgeInterface) error {
	if !config.EnableIPv6 {
		return nil
	}
	// .....	
}

Reply:
Maybe.  This test code is mimicking the following:
https://github.com/moby/moby/blob/51d647122ad34628a92252c2b2a9f865f4b20613/libnetwork/drivers/bridge/bridge_linux.go#L743-L747
If `setupBridgeIPv6` is changed then we should change that code too, and I think that deserves its own consideration in a separate pull request.